test/check-application

```diff
@@ -117,7 +117,8 @@ class TestApplication(testlib.MachineCase):
 
         cert_name = "Server-Cert"
         db_path = "/etc/pki/nssdb"
-        m.execute("selfsign-getcert request -d {0} -n {1}".format(db_path, cert_name))
+        m.execute("selfsign-getcert request -d {0} -n {1} -I {2}".format(db_path, cert_name, cert_name))
```

Comment:
This commit says that you made `testModifyAutorenewal` non destructive, yet you did not use `@nond` decorator

Reply:
Also you don't need to repeat `cert_name`, just `m.execute("selfsign-getcert request -d {0} -n {1} -I {1}".format(db_path, cert_name))` would do.